### PR TITLE
[SC-159] Add README

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,12 @@
...
+
+The codebase uses [Foundry](https://github.com/foundry-rs/foundry) as the development framework. In order to run tests, deploy contracts, or perform other operations, use standard [Foundry](https://github.com/foundry-rs/foundry) commands.
+
+## ⚙️ Components
+### 🔊 Forwarders
+Forwarders are contracts on the host domain, where the governance of the protocol resides. <br>These contracts abstract away all complexity of using a bridge and facilitate seamless message passing between domains. <br>Host domain admin uses forwarders to trigger payload executions on a bridged domain instances of the protocol.
+### 🚦 Executors
+Executors serve as admins of the bridged domain instances of the protocol. <br>They are responsible for storying the queue of proposals passed from the host domain governance and their execution. <br> They manage bridged domain protocol instance using standard payload pattern.
+## ✍️ Architecture Diagram
+![Architecture Diagram](/diagram.png)
```
Comment (hexonaut) [inline README.md:12]: This diagram is not general. The validator set is specific to Gnosis Chain and this does not necessarily exist for other chains such as L2s.

Comment (barrutko) [reply README.md:12]: There has to be some EOA-ish type of entity that initiates the execution on the L2 side and that's what I wanted to picture here. I'm in favour of renaming and keeping it, but I can remove as well. Any suggestions?

Comment (hexonaut) [reply README.md:12]: I would call it "Relayer" then or remove it.

Comment (barrutko) [reply README.md:12]: Ok, I removed that part actually.

**Review by hexonaut — APPROVED** [review]
LGTM for now. We probably want more details about how to contribute in a subsequent PR.

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**